.travis.yml

```diff
@@ -62,7 +68,7 @@ before_script:
 
 script:
   - composer validate
-  - phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+  - vendor/bin/simple-phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
```

Comment:
Didn't see that but why using simple-phpunit ?

And in this case, phpunit should be remove form the global require: https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge/pull/81/files#diff-354f30a63fb0907d4ad57269548329e3R61